src/test/java/ru/ayaz/CommentTest.java

```diff
@@ -0,0 +1,49 @@
+package ru.ayaz;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+
+public class CommentTest {
+
+    @Test
+    public void shouldBeEqualWhenReferencesAreSame() {
+        Comment comment = new Comment("Ayaz", "Nice video!");
+        Comment anotherComment = comment;
+
+        assertTrue(comment.equals(anotherComment));
+    }
+
+    @Test
+    public void shouldNotBeEqualWhenInstancesAreDifferent() {
+        Comment comment = new Comment("Ayaz", "Nice video!");
+        Object object = mock(Video.class);
+
+        assertFalse(comment.equals(object));
+    }
+
+    @Test
+    public void shouldNotBeEqualWhenAuthorsAreDifferent() {
```

Comment:
Well, you are probably looking from pure TDD perspective on the problem - that is write test, make it fail, write the simplest possible implementation that passes the test, repeat ...

It is just a general guideline which doesn't apply always.
E.g. here you wrote 1 test, then auto generated implementation for "equals". This implementation was not the simplest possible, so, it predicts other yet untested cases.
So, all your tests are covered by functionality (otherwise they won't pass); If you would have written the simplest functionality possible then all your functionality would have been covered by tests. But you didn't write the simplest implementation. You auto generated it. And now not all your functionality is covered by tests.
Doesn't the suggested test add code coverage at all ? (provided you corrected all other issues)

Even if it doesn't, I suggested this test purely from documentation perspective.
It is a use-case which might not be very clear (in this particular case its quite clear that comments should differ, but in other domains that might not be as clear).